src/services/decoders/utils.ts

```
@@ -0,0 +1,62 @@
+import {
+    ParsedKey,
+    REPLACEMENT_CHARACTER,
+} from "../../typings/formatters";
+import {
+    parseKey,
+    replaceDoubleBacklash,
+} from "../formatters/YscopeFormatter/utils";
+import {
+    DecoderOptions,
+} from "../../typings/decoders";
```

ESLint

[error] Imports must not be broken into multiple lines if there are 1 or less elements.

(import-newlines/enforce)